app-i18n/fcitx-qt/files/fcitx-qt-5.1.3-backport-conditional-wayland.patch

```diff
@@ -0,0 +1,155 @@
+Backport upstream commit af535e7e making QtWayland optional.
```

Comment:
It would be good to include links to upstream commits, if needed in future it will be easier to find how and where it's taken from.